Lib/fontTools/varLib/cff.py

```diff
@@ -157,17 +157,19 @@ def get_private(regionFDArrays, fd_index, ri, fd_map):
 
 def merge_PrivateDicts(top_dicts, vsindex_dict, var_model, fd_map):
 	"""
-	I step through the FontDicts in the FDArray of the varfont TopDict.
-	For each varfont FontDict:
-		step through each key in FontDict.Private.
-		For each key, step through each relevant source font Private dict, and
+	I step through the ``FontDicts`` in the ``FDArray`` of the varfont ``TopDict``.
```

Comment:
Documenting a function in first person from the function’s point of view is stylistically inconsistent with the rest of the documentation.

Reply:
Yep, again I haven't got to this systematically but just wanted to fix up an RST formatting warning. Will fix in part 2.